vertx-web/src/test/java/io/vertx/ext/web/handler/CORSHandlerTest.java

```diff
@@ -216,16 +224,6 @@ public void testChaining() throws Exception {
     assertSame(cors, cors.exposedHeaders(new HashSet<>()));
   }
 
-  @Test
-  public void testUnsecureCorsShouldNotBeAllowed() throws Exception {
```

Comment:
I think all browsers don't allow this combination, it should be tested.

Reply:
That should not be an issue for the browser, because browser sends only the origin header and, in this particular case, the server replies with allowed origin header as echo of origin header and allowed credentials header true